### PR TITLE
Add Basic Structure and Components 🚀

### PR DESCRIPTION
**Pull request by hsztan**
I create a basic CRA boilerplate plate 🍛 and eliminate unnecessary elements. I then make the necessary components ⚙️ for the bookstore container 📚 list and the necessary pages 📑 to navigate between routes. I also prepare the book 📔 component to receive and validate ✅ props to receive data. 😎

### CODE REVIEW

File: src/pages/AuthorsPage.jsx [hunk 1]
```diff
@@ -0,0 +1,8 @@
+const AuthorsPage = () => (
+  <div>
+    <h1>THis is the Authors Page</h1>
+    <button type="button">Check status</button>
+  </div>
+);
+
+export default AuthorsPage;
```
Comment (nicupop729) [inline src/pages/AuthorsPage.jsx]: In order to respect the logic that this project follows, we need to respect the milestone requirements regarding the names of components and the name of the different sections of our app.

- [x] Kindly, change the name to this component and file from AuthorsPage to Categories and, kindly, respect the requirements of the project (in future we will have to respect the requirements set by clients) :+1: 

Screenshot from project requirements:
image

File: src/App.js [hunk 2]
```diff
@@ -1,10 +1,18 @@
...
 
 function App() {
   return (
     <div className="App">
       <header className="App-header">
-        <h1>Hello World</h1>
+        <NavBar />
+        <Routes>
+          <Route path="/" element={<Home />} />
+          <Route path="/authors" element={<AuthorsPage />} />
```
Comment (nicupop729) [inline src/App.js]: In order to respect the logic that this project follows, we need to respect the milestone requirements regarding the names of components and the name of the different sections of our app.

- [x] Kindly, change the path name from AuthorsPage to Categories;

Screenshot from project requirements:
image

**Review by nicupop729 — CHANGES_REQUESTED** [review]
Hi @hsztan, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
 - [X] Correct git flow practice :heavy_check_mark:
 - [X] Good commit messages :heavy_check_mark:
 - [X] Professional README :heavy_check_mark:
 - [X] Linters are passing :heavy_check_mark:
 - [X] Clean codebase :heavy_check_mark:

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/App.js (hunk 2 above)
Comment (hsztan) [reply src/App.js]: @nicupop729 thank you for your feedback. I will change it per requirements ASAP 🚀

**Review by andersonlebon — APPROVED** [review]
## `Hi @hsztan`

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 :clap: :clap: :100: 

nodding-yes

## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Good PR description :heavy_check_mark: 
- Your README file is now professional :heavy_check_mark: 

## Optional suggestions
- [ ] Kindly Check the comments under the review._
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**